core/sphinx/geo/geo.go

```diff
@@ -0,0 +1,327 @@
+package geo
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/BurntSushi/toml"
+	"github.com/fxamacker/cbor/v2"
+	"golang.org/x/crypto/sha3"
+
+	"github.com/cloudflare/circl/kem"
+	kemschemes "github.com/cloudflare/circl/kem/schemes"
+
+	"github.com/katzenpost/katzenpost/core/crypto/nike"
+	"github.com/katzenpost/katzenpost/core/crypto/nike/schemes"
+	"github.com/katzenpost/katzenpost/core/sphinx/constants"
+	"github.com/katzenpost/katzenpost/core/sphinx/internal/crypto"
+)
+
+const (
+	surbReplyLength = 1 + constants.SURBIDLength
```

Comment:
I'm assuming the `1 +` in these constants are 1-byte type tags? could we put a comment explaining what they are? (and perhaps the name of the function that decodes them?)

Reply:
okay i've added another sphinx constant for that

Reply:
you added a new constant but aren't using here on line 23 yet